modules/openapi-generator/src/test/resources/2_0/issue12219.yaml

```diff
@@ -0,0 +1,53 @@
+swagger: '2.0'
+info:
+  description: 'blah'
+  version: 1.0.0
+  title: sample spec
+host: fake.site.com
+tags:
+  - name: Test
+schemes:
+  - https
+paths:
+  /test:
+    post:
+      summary: Post to test
+      description: ''
+      operationId: postToTest
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - in: body
+          name: Obj to test
+          schema:
+            $ref: '#/definitions/ObjTest'
+      responses:
+        '201':
+          description: successful operation
+          schema:
+            $ref: '#/definitions/ObjTest'
+      x-operation-extra-annotation: '@javax.annotation.security.RolesAllowed({"ROLE_TEST"}) @org.springframework.security.access.annotation.Secured({"ROLE_TEST"}) @org.springframework.security.access.prepost.PreAuthorize("hasRole(''ROLE_TEST'') and hasRole(''ROLE_TEST_TWO'')")'
```

Comment:
add onle more test with array like style(which I believe would be used widely)

```yaml
x-operation-extra-annotation: 
  - @javax.annotation.security.RolesAllowed({"ROLE_TEST"})
  - @org.springframework.security.access.annotation.Secured({"ROLE_TEST"})
  - @org.springframework.security.access.prepost.PreAuthorize("hasRole('ROLE_TEST') and hasRole('ROLE_TEST_TWO')")
```

Reply:
done